docs/articles/lwc/localdev.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Local Development Server (Beta)
```

Comment:
Same, "Local Development", the feature is local development, the implementation is the "Local Development Server"